src/utilities/_themes.scss

```diff
@@ -0,0 +1,50 @@
+/* Alpha Theme */
```

Comment:
Does this work? Where do you import the scss variables?

Reply:
The scss variables are now imported, but he functionality of themes still does not work.

Reply:
I'll look into it at the next stable version.